### PR TITLE
decode-morse-message

### PR DESCRIPTION
**Pull request by Kgomotso196**
# PROJECT TO DOs
### IN THIS PROJECT WE:
1. Used Ruby syntax to decode Morse message.
2. Used Ruby syntax for basic programming operations.
3. Ran a program using the command line.
4. Made sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
5. Made sure that we used [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md).
6. Made sure that we documented our work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).
7. Created a method to decode a Morse code character, takes a string parameter, and return the corresponding character in uppercase (e.g. `decode_char(".-")` returns `"A"`).
8. Created a method to decode an entire word in Morse code, takes a string parameter, and return the string representation. Every character in a word is separated by a single space (e.g. `decode_word("-- -.--")` returns `"MY"`).
9. Created a method to decode the entire message in Morse code, takes a string parameter, and return the string representation. Every word is separated by 3 spaces (e.g.
>     decode("-- -.--   -. .- -- .")
returns `"MY NAME"`).

### CODE REVIEW

**Review by rloterh — COMMENTED** [review]
Hi Team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Let's ensure that all assigned members to this project contribute evenly to this task. One member working on only the ReadMe is not a fair contribution to the task.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1,291 +1,173 @@
...
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+👤 **Author**
 
-👤 **Author2**
+- GitHub: [@yemidada](https://github.com/yemidada)
+- Twitter: [@naijavine](https://twitter.com/naijavine)
+- LinkedIn: [yemidada](https://linkedin.com/in/yemidada)
```
Comment (OyePriscilla) [inline README.md]: - [ ] Please kindly fix the twitter account link, I got a message that the account does not exist. If you don't have a working link at the moment, I think it's better to remove it in order to have an engaging README file. ℹ️

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @Kgomotso196 and @yemidada 👍 👍, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
Created a method to decode a Morse code characters ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @Kgomotso196, and @yemidada👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- Good job with the Decode Morse message. ✔️
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**